dotcom-rendering/makefile

```diff
@@ -64,13 +50,13 @@ dev: clear clean-dist install
 
 percy: clear clean-dist install
 
-cypress: clear clean-dist install
-	$(call log, "starting frontend DEV server for Cypress")
-	@NODE_ENV=development start-server-and-test 'node scripts/frontend/dev-server' 3030 'cypress run --spec "cypress/integration/**/*"'
+cypress: clear clean-dist install build
```

Comment:
To be clear, is this just for testing locally? It looks like the Github Action runs Cypress in CI

Reply:
Yes, `make cypress` is just for local execution as it doesn't allow for matrix execution

Reply:
Actually, it's also used by Team City in [ci-dcr.sh](https://github.com/guardian/dotcom-rendering/blob/356acb5ad77fb417202a8f9e7febdc6a854ccebf/scripts/ci-dcr.sh)